docs/OV_Runtime_UG/supported_plugins/CPU.md

```diff
@@ -44,14 +47,14 @@ Supported Inference Data Types
 CPU plugin supports the following data types as inference precision of internal primitives:
 
 | - Floating-point data types:
-|   - f32
-|   - bf16
+|   - f32 (Intel® x86-64, Arm®)
+|   - bf16 (Intel® x86-64)
 | - Integer data types:
-|   - i32
+|   - i32 (Intel® x86-64, Arm®)
 | - Quantized data types:
-|   - u8
-|   - i8
-|   - u1
+|   - u8 (Intel® x86-64, Arm®)
```

Comment:
We do not use u8 on ARM. Should be deleted.

Reply:
Fixed